samples/editors/date-picker/date-limits/App.razor

```diff
@@ -0,0 +1,24 @@
+@using IgniteUI.Blazor.Controls
+@inject IIgniteUIBlazor IgniteUIBlazor
+
+<div class="container vertical">
```

Comment:
add a DIV with class="options horizontal" and 2 labels showing values of MinDate and MaxDate

Reply:
pending